apis/hubspot/crm/helpers/getMe.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+from hubspot import HubSpot
+
+token = os.getenv("GPTSCRIPT_API_HUBAPI_COM_BEARER_TOKEN")
+client = HubSpot(access_token=token)
+
+
+def get_me():
+    try:
+        user = client.auth.oauth.access_tokens_api.get(token)
+        print(f"Current user is {user.user}. UserId is {user.user_id}.")
+    except Exception as e:
+        print(f"Error getting current user information: {e}")
```

Comment:
nit: I think we should exit(1) after this

```suggestion
    except Exception as e:
        print(f"Error getting current user information: {e}")
        exit(1)
```